README.md

```diff
@@ -1 +1,50 @@
-# Trombones-Landing-page
\ No newline at end of file
+# Trombones-Landing-page
+This is a practice project about creating a Landing page of an Original Trombones business using html and css files.
+
+## Built With
+
+- Html and Css
+- No Frameworks
+
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+Download this GitHub Repo and go the index.html file
+### Install
+Set up liveserver as an extension in your VS Code
+### Usage
+Right click and open the index.html in your browser to view
+
```

Comment:
- [ ] Your comments should be clear and easy to understand. For Example:

* This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
* [This](https://github.com/medaminedev66/Trombones-Landing-page/pull/1#issuecomment-875425762) is a vague comment: *Try to add more information on the Readme file, and fix the linters issues*